### PR TITLE
Add `Packages.build_recursive`; shortwire already built packages

### PR DESCRIPTION
**Pull request by lgoettgens**
Extracted from https://github.com/oscar-system/GAP.jl/pull/1067 since it is no longer needed there.

### CODE REVIEW

**Comment by fingolfin** [comment]
Tiny nitpick: "already build" -> "already built"

**Comment by lgoettgens** [comment]
> Tiny nitpick: "already build" -> "already built"

In the PR title? Fixed

File: CHANGES.md [hunk 1]
```diff
@@ -31,6 +31,9 @@
   - orb
   - profiling
   - simpcomp
+- Add `GAP.Packages.build_recursive(name)`
+- `GAP.Packages.build(name)` no longer tries to build the package if it is
+  already installed
```
Comment (fingolfin) [inline CHANGES.md:36]:
```suggestion
- Change `GAP.Packages.build(name)` to no longer try to build the package if
  it is already installed
```

**Review by fingolfin — APPROVED** [review]
Looks good to me. One minor nitpick.

Of course it'd be great if we had a test for this, but that can come at another time. (As long as someone at least tests it manually, once?)

**Comment by lgoettgens** [comment]
> Of course it'd be great if we had a test for this, but that can come at another time. (As long as someone at least tests it manually, once?)

A previous iteration of #1067 used to use this function, so it worked a week ago

**Pull request merged**